docs/jmeter/README.md

```diff
@@ -65,9 +65,10 @@ JMETER_WORKER_MEMORY_REQUESTS
 
 For the full list of possible environment variables check [Kangal environment variables](../env-vars.md)
 
-More information regarding resource limits and requests can be found in the following page(s):
-- https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
-- https://cloud.google.com/blog/products/gcp/kubernetes-best-practices-resource-requests-and-limits
+More information regarding resource limits and requests can be found in the following pages:
+
+- [https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/](https://github.com/hellofresh/kangal/tree/master/examples)
```

Comment:
I believe the url is wrong no? Also, for the next line.

Reply:
Good catch! Wrong url in copy buffer 😅